<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWorkspaceWidget</class>
 <widget class="QWidget" name="QWorkspaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QHttpMethodCombo" name="httpMethodCombo"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtUrl"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnSend">
        <property name="text">
         <string>Send</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/img/send.png</normaloff>:/img/send.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QToolButton" name="btnAddHeader">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="3" rowspan="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>303</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" rowspan="2" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Headers</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QHttpHeaderTable" name="tableHeaders"/>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="btnRemoveHeader">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Body</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="btnPasteBody">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/img/paste.png</normaloff>:/img/paste.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>274</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCodeTextEdit" name="txtBody"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" rowspan="2" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="text">
          <string>Response</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>780</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QCodeTextEdit" name="txtResponse"/>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="btnCopyResponse">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../res.qrc">
           <normaloff>:/img/copy.png</normaloff>:/img/copy.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="btnClearResponse">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../res.qrc">
           <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHttpMethodCombo</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/qhttpmethodcombo.h</header>
  </customwidget>
  <customwidget>
   <class>QHttpHeaderTable</class>
   <extends>QTableWidget</extends>
   <header location="global">widgets/qhttpheadertable.h</header>
  </customwidget>
  <customwidget>
   <class>QCodeTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">widgets/codeeditor/qcodetextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
